feconf.py

```diff
@@ -1147,6 +1147,10 @@ def get_empty_ratings():
 ROLE_VIEWER = 'viewer'
 ROLE_NONE = 'none'
 
+# The list of entity types that do not require entity specific access control
+# when viewing respective suggestions.
+ENTITY_TYPES_WITH_UNRESTRICTED_VIEW_SUGGESTION_ACCESS = [ENTITY_TYPE_SKILL]
```

Comment:
Can you explain how/where is this used?

Reply:
Sure. We use it in the ACL layer like this:

https://github.com/oppia/oppia/blob/abe5926dbf278fe639a62640bbb86f3d26693e91/core/controllers/acl_decorators.py#L804-L808

Basically, the logic following this part is specific to suggestions in explorations (which require some validation of access rights). For viewing feedback in case of translations, it is fine to execute the "access control" logic because translations belong to explorations but this logic will error for other types of suggestions that are not exploration specific (e.g. questions). Such suggestion entities need to be listed in `ENTITY_TYPES_WITH_UNRESTRICTED_VIEW_SUGGESTION_ACCESS` so that we skip the checks.